examples/nextjs/README.md

```diff
@@ -0,0 +1,48 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with
+[`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+
+## Getting Started
```

Comment:
Before running the dev server, it is necessary to install the packages, right? I tried with npm install, but I got an error:

```
% npm i                                                                                                                                                                                                                    
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:*
```

But it worked with pnpm: `pnpm install`

Should we add `pnpm install` to this section?